cmd/schemagen/docsgen/docsgen/templates/header.tmpl

```diff
@@ -0,0 +1,14 @@
+# {{ packageName .Type }} Configuration Reference
```

Comment:
Yes, this is appears to be doable. I also wanted this, but thought it wasn't possible because we only have an instance of the Config at runtime, but just realized we can get this info out of the factories (from which we get the Config) and pass those along in a map or something.

Reply:
I made this `Otlp Receiver Reference`. Can change it to `"otlp" Receiver Reference` if you'd prefer.